editor/store-persist.js

```diff
@@ -41,7 +47,7 @@ export default function storePersist( reducerKey, storageKey = DEFAULT_STORAGE_K
 
 			store.dispatch( {
 				type: 'REDUX_REHYDRATE',
-				payload: persistedState,
+				payload: beforeReHydrate( persistedState ),
```

Comment:
Personal preference would be to capitalize as `beforeRehydrate`, since it doesn't seem that Re and Hydrate are separate words.

Reply:
Corrected 👍